### PR TITLE
[DQT] Restore help file; update contents

### PR DESCRIPTION
**Pull request by johnsaigle**
* Replaces #5200 

linked to: #5576

### CODE REVIEW

**Comment by johnsaigle** [comment]
Cool, yeah I'm not sure about those changes either...

**Comment by johnsaigle** [comment]
Adding Bug here because this module will not have any help text without this PR

**Comment by christinerogers** [comment]
How about tagging the person who reactified / updated the module, for their comments?

**Comment by johnsaigle** [comment]
I think Jordan was the last person to seriously work on it... not sure who else to contact.

**Comment by christinerogers** [comment]
I took a closer look, @knoll-alex  @johnsaigle  - I think this is very fix-able without dev input - here's how : 

> ... it looks like there have been changes to the module since I wrote this help text

Good catch!  Though these changes are quite old, I wonder how we didn't caught them in the first round of review. 
That said, they're minor, and I think can be easily tackled with 2 quick tweaks to this PR : 

> saved queries tabs are gone

- [ ] This is now  `Load saved queries` and `Manage Saved Queries`. if the functionality isn't very simliar / self-evident based on the old help text, tag me on your next questions in your next commit. 

> statistical analysis tab has been added

- [ ] "This tab shows basic statistical calculations and scatterplot visualization of data from your query." is probably all we need to say. 

Also - how to use it: 

> (I don't see any functionality in the statistics tab currently)

To see the functionality, load a Saved query like `Sotirios_test_q`, run it in the `View Data` tab, and then click to the Statistical Analysis tab.  (select data fields for the scatterplot).  
I think this follows the natural flow of the help text for the module, so not sure it's worth re-explaining that you need a query loaded.

**Comment by johnsaigle** [comment]
@christinerogers @knoll-alex You should be able to commit directly to this PR or use the `suggest` feature to make changes so please feel free to do so.

**Comment by knoll-alex** [comment]
Ahhh, my apologies, I forgot that the Saved Queries tabs are all the way to the right side and I just didn't notice them. I'll make the changes.

**Comment by knoll-alex** [comment]
Thanks Christine and John, this is done. I think you can remove the Bug tag.

File: modules/dataquery/help/dataquery.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+To add a filter, select an instrument using the dropdown. A secondary dropdown will appear, listing all data fields in that category. Once a data field is selected, additional dropdowns to specify the “Operator” and “Value” will appear. These _Operators_ are:
+
+   = equal to<br>
+   != does not equal<br>
+   <= less than or equal to<br>
+   \>= greater than or equal to<br>
+   _startsWith_ : must begin with a specific character or string<br>
+   _contains_ : must contain a specific character or string<br>
+
```
Comment (PapillonMcGill) [inline modules/dataquery/help/dataquery.md]: Missing info for `isNull` and `isNotNull`.

Comment (christinerogers) [reply modules/dataquery/help/dataquery.md]: ok ill make a commit.

Comment (christinerogers) [reply modules/dataquery/help/dataquery.md]: ready for re-review @PapillonMcGill

**Comment by christinerogers** [comment]
ready to merge for the 22 release  @driusan

**Pull request merged**